src/plugins/orangehrmDashboardPlugin/config/routes.yaml

```diff
@@ -16,9 +16,9 @@ apiv2_dashboard_employees_by_subunit:
 #       Mock API
 ##########################
 
-apiv2_dashboard_employees_subunit_distribution:
-  path: /api/v2/dashboard/employees/subunit
-  controller: OrangeHRM\Dashboard\Controller\DashboardDataMockController::getEmployeeSubunitDistribution
+apiv2_dashboard_employees_location_distribution:
```

Comment:
@ManulMax Please remove this route and DashboardDataMockController.php when you're finishing the location API